llvm/lib/YkIR/YkIRWriter.cpp

```diff
@@ -239,6 +239,30 @@ class YkIRWriter {
     InstIdx++;
   }
 
+  void serialiseCallInst(CallInst *I, ValueLoweringMap &VLMap, unsigned BBIdx,
+                         unsigned InstIdx) {
+    // type_index:
+    OutStreamer.emitSizeT(typeIndex(I->getType()));
+    // opcode:
+    serialiseOpcode(OpCode::Call);
+    // num_operands:
+    unsigned NumOpers = I->getNumOperands();
+    OutStreamer.emitInt32(NumOpers);
+
+    // OPERAND 0: What to call.
+    //
+    // In LLVM IR this is the final operand, which is a cause of confusion.
+    serialiseOperand(I, VLMap, I->getOperand(NumOpers - 1));
```

Comment:
If `NumOpers` gives the number of operands, and the index starts at zero, shouldn't the last operand of the call be `NumOpers - 2`?

Reply:
No, I'm talking bollocks. Ignore me.